specs/waku/v2/waku-v2.md

```diff
@@ -122,7 +122,7 @@ See [bridge spec](waku-bridge.md) for details on a bridge mode.
 
 ## Changelog
 
-### 2.0.0-beta1
+### 2.0.0-beta2
```

Comment:
Same here, also see above re changelog. We don't want to overwrite it, the goal is to have a short history to see what changed etc between versions.

See https://specs.vac.dev/specs/waku/waku.html#changelog for an example